modules/qr_input.py

```diff
@@ -0,0 +1,38 @@
+"""
+Function to read camera input until valid QR code
+"""
+
+from common.qr.modules import qr_scanner
+from common.camera.modules import camera_device
+
+
+def qr_input(device: "int | string") -> str:
```

Comment:
Change the return type to `tuple[bool, str | None]` , since the user may quit before QR text is found.